app/views/admin/loans/_loans_grid.html.slim

```diff
@@ -8,11 +8,12 @@
     - g.column name: t('common.id'), attribute: 'id', filter: true, in_csv: false, detach_with_id: :id_filter do |loan|
       - link_to(loan.id, admin_loan_path(loan.id))
     - g.column name: csv_id, attribute: 'id', in_html: false, filter: false
-    - g.column name: t('activerecord.attributes.loan.name'), attribute: 'name',
-      in_html: false, filter: false
+
+    - g.column name: t('activerecord.attributes.loan.name'), attribute: 'name' do |loan|
+      - loan.display_name
```

Comment:
I am confirming loan.display_name shows up in both the loan index page and an export of the loans data.